Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -193,223 +193,223 @@
-    <string name="information_about_headline">Gemeinsam Corona bekämpfen</string>
-    <string name="information_privacy_title">Datenschutzinformation</string>
+    <string name="information_about_title">About the app</string>
+    <string name="information_about_headline">Fight Corona together</string>
+    <string name="information_privacy_title">Data protection information</string>
     <string name="information_privacy_headline">@string/lorem_ipsum</string>
-    <string name="information_terms_title">Nutzungsbedingungen</string>
+    <string name="information_terms_title">Terms of use</string>
     <string name="information_terms_headline">@string/lorem_ipsum</string>
-    <string name="information_contact_title">Kontakt</string>
-    <string name="information_contact_headline">Wie können wir Ihnen helfen?</string>
-    <string name="information_help_title">Häufige Fragen</string>
-    <string name="information_technical_title">Rechtliche Hinweise</string>
+    <string name="information_contact_title">Contact</string>
+    <string name="information_contact_headline">How can we help you?</string>
+    <string name="information_help_title">Frequently asked questions</string>
+    <string name="information_technical_title">Legal information</string>
     <string name="information_technical_headline">FTP/SFTP/FTPS client uses libs</string>
-    <string name="information_legal_title">Impressum</string>
-    <string name="information_legal_headline">Herausgeber</string>
-    <string name="information_legal_body">SAP Deutschland SE &amp; Co. KG \n \nHauptsitz: \nSAP Deutschland SE &amp; Co. KG \nHasso-Plattner-Ring 7 \n69190 Walldorf </string>
+    <string name="information_legal_title">Imprint</string>
+    <string name="information_legal_headline">Publisher</string>
+    <string name="information_legal_body">SAP Deutschland SE &amp; Co. KG \n \nHeadquarter: \nSAP Deutschland SE &amp; Co. KG \nHasso-Plattner-Ring 7 \n69190 Walldorf </string>
 
     <!-- ####################################
              Submission
              TODO: rename if screens are available
     ###################################### -->
     <!-- Permission Rationale Dialog -->
-    <string name="submission_qr_code_scan_permission_rationale_dialog_headline">Kamera Berechtigung benötigt</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_headline">Camera permission required</string>
     <string name="submission_qr_code_scan_permission_rationale_dialog_body">@string/lorem_ipsum</string>
 
-    <string name="submission_qr_code_scan_permission_rationale_dialog_button_positive">Erlauben</string>
-    <string name="submission_qr_code_scan_permission_rationale_dialog_button_negative">Nicht erlauben</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_button_positive">Allow</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_button_negative">Not allow</string>
```

Comment:
Can this be just "Cancel" or "Don't allow"?

Reply:
Fixed. Changed it to "Don't allow"